examples/guess-the-sketch/src/app.py

```diff
@@ -213,5 +209,21 @@ def handle_message(data):
             # Send the first picture to the opponent for guessing
             emit('guess_sketch_response', {'image': player_prompt[player][1]['picture'], 'prompt': player_prompt[player][1]['prompt'], 'opponentId': player, 'round': 1}, room=player_id)
 
+def similarity(p1, p2):
+    '''Calculate dot product distance between two prompts'''
+
+    resp = requests.post(
```

Comment:
Does this need error handling? What if the embeddings endpoint is down?

Reply:
line 220 :)

Reply:
`raise_for_status` is just a convenience function that says "raise an Exception if it's anything other than `200`.

Reply:
I'm wondering when the exception gets raised what happens? It bubbles up to the calling function which .... crashes? Does anything come back to the UI?